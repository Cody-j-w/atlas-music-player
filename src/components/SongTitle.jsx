export default function SongTitle(props) {
    return (
        <div>
            <h2 className="text-3xl text-charcoal dark:text-bright-red font-bold my-3">{props.songTitle}</h2>
            <p className="text-xl text-charcoal dark:text-silver font-semibold">{props.artist}</p>
        </div>
    )
}