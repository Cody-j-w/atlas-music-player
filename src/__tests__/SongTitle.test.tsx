import SongTitle from "../components/SongTitle";
import { test, expect } from "vitest";
import { render } from "@testing-library/react";

test('SongTitle snapshot', () => {
    const { container } = render(<SongTitle songTitle="test One" artist="The Tests" />);
    expect(container).toMatchSnapshot();
});