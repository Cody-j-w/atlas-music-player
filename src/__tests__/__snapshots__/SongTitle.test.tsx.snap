// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`SongTitle snapshot 1`] = `
<div>
  <div>
    <h2
      class="text-3xl text-charcoal dark:text-bright-red font-bold my-3"
    >
      test One
    </h2>
    <p
      class="text-xl text-charcoal dark:text-silver font-semibold"
    >
      The Tests
    </p>
  </div>
</div>
`;
